src/python/turicreate/toolkits/recommender/__init__.py

```diff
@@ -106,6 +108,7 @@
 item_similarity_recommender = _LazyModuleLoader(_mod_par + 'item_similarity_recommender')
 item_content_recommender = _LazyModuleLoader(_mod_par + 'item_content_recommender')
 
-# TODO: lazy callable
-from . import util
-from .util import _create as create
+# from . import util
```

Comment:
Let's remove these commented out lines of code.

Reply:
oh yeah